definitions.yml

```
@@ -412,13 +412,14 @@ definitions:
       program_active:
         description: "If true, the cashback program is active and all the related in app functionalities are enabled"
         type: boolean
-  ZendeskConfig:
+  AssistanceToolConfig:
     type: object
     description: A specific config for Zendesk
     required:
-      - active
+      - tools
```

```suggestion
      - tool
```
It's more appropriate for a single value

---

We should also update the `description`

`description: A specific config for Zendesk`